README.md

```diff
@@ -248,12 +249,53 @@ You will need a keystore that contains the server certificate. The bare minimum
  keytool -genkey -keyalg RSA -alias jobserver -keystore ~/sjs.jks -storepass changeit -validity 360 -keysize 2048
 ```
 You may place the keystore anywhere.    
-Here is an example of simple curl command that utilizes ssl:
+Here is an example of a simple curl command that utilizes ssl:
 ```
 curl -k https://localhost:8090/contexts
 ```
 The ```-k``` flag tells curl to "Allow connections to SSL sites without certs". Export your server certificate and import it into the client's truststore to fully utilize ssl security. 
 
+### Authentication
+
+Authentication uses the [Apache Shiro](http://shiro.apache.org/index.html) framework. Authentication is activated by setting this flag (Section 'shiro'): 
+```
+authentication = on
+# absolute path to shiro config file, including file name
+path.ini = "/some/path/shiro.ini"
+```
+Shiro-specific configuration options should be placed into a file named 'shiro.ini' in the directory as specified by the config option 'path.ini'. 
+Here is an example that configures LDAP with user group verification:
+```
+# use this for basic ldap authorization, without group checking
+# activeDirectoryRealm = org.apache.shiro.realm.ldap.JndiLdapRealm
+# use this for checking group membership of users based on the 'member' attribute of the groups:
+activeDirectoryRealm = spark.jobserver.auth.LdapGroupRealm
+activeDirectoryRealm.contextFactory.environment[java.naming.security.credentials] = password
+activeDirectoryRealm.contextFactory.url = ldap://xxx.yyy..org:389
+activeDirectoryRealm.userDnTemplate = cn={0},ou=people,dc=xxx,dc=org
+
+cacheManager = org.apache.shiro.cache.MemoryConstrainedCacheManager
+
+securityManager.cacheManager = $cacheManager
+```
+
+The group verification currently requires two more settings (Section 'shiro' in your application.conf file):    
+```
+ldap {
+    # indicates that only users that are in either 'group1' or 'group2' or both are admitted
+    # uncomment only if checks on groups are desired and make sure to adjust shiro.ini accordingly
+    # an empty list of allowed groups also effectively disables group checking
+	allowedGroups = ["cn=group1,ou=groups", "cn=group2,ou=groups"]
+    # ldap search base
+    searchBase = "dc=xxx,dc=org"
+}
+```
+
+Here is an example of a simple curl command that authenticates a user and uses ssl:
+```
+curl -k --basic --user 'user:pw' https://localhost:8090/contexts
```

Comment:
Not a biggie, but I prefer to directly pass in the basic authentication token header using -H so that plain password won't be exposed.